backend/cmd/api/main.go

```diff
@@ -87,6 +108,13 @@ func main() {
 
 	log.L.Debug().Msgf("connect to http://localhost:%s/ for GraphQL playground", config.Port)
 
+	searchService := search.New(config.Algolia.AppId, config.Algolia.ApiKey, db)
+	configureSearchEndpoints(r, config, &searchService)
```

Comment:
We should talk about this. I have considered doing things this way and my conclusion in the end was that all of endpoint config should happen in the main function. The main reason for it is that when you are looking for it in 6 months you want to have one place to look.